addons/l10n_ad/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
Know that all these Part of Odoo.... can be removed (but I won't block if you prefer keeping them)